yoti_python_sdk/dynamic_sharing_service/policy/dynamic_policy_builder.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from yoti_python_sdk import config
+
+from .wanted_attribute_builder import WantedAttributeBuilder
+
+"""
+Builder for DynamicPolicy
+"""
+
+
+class DynamicPolicyBuilder(object):
+    SELFIE_AUTH_TYPE = 1
+    PIN_AUTH_TYPE = 2
+
+    def __init__(self):
+        self.__wanted_attributes = {}
+        self.__wanted_auth_types = {}
+        self.__is_wanted_remember_me = False
+
+    """
+    @param wanted_attribute
+    """
+
+    def with_wanted_attribute(self, wanted_attribute):
+        key = (
+            wanted_attribute["derivation"]
+            if wanted_attribute.get("derivation", False)
+            else wanted_attribute["name"]
+        )
+        self.__wanted_attributes[key] = wanted_attribute
+        return self
+
+    """
+    @param wanted_name The name of the attribute to include
+    """
+
+    def with_wanted_attribute_by_name(self, wanted_name):
+        attribute = WantedAttributeBuilder().withName(wanted_name).build()
+        return self.with_wanted_attribute(attribute)
+
+    def with_family_name(self):
+        return self.with_wanted_attribute_by_name(config.ATTRIBUTE_FAMILY_NAME)
+
+    def with_given_names(self):
+        return self.with_wanted_attribute_by_name(config.ATTRIBUTE_GIVEN_NAMES)
+
+    def with_full_name(self):
+        return self.with_wanted_attribute_by_name(config.ATTRIBUTE_FULL_NAME)
+
+    def with_date_of_birth(self):
+        return self.with_wanted_attribute_by_name(config.ATTRIBUTE_DATE_OF_BIRTH)
+
+    def with_age_derived_attribute(self, derivation):
+        attribute = (
+            WantedAttributeBuilder()
+            .withName(config.ATTRIBUTE_DATE_OF_BIRTH)
+            .withDerivation(derivation)
+            .build()
+        )
+        return self.with_wanted_attribute(attribute)
+
+    def with_age_over(self, age):
+        return self.with_age_derived_attribute(config.ATTRIBUTE_AGE_OVER + str(age))
```

Comment:
Since Python is dynamically typed, could we also put a check here (and in the method below) that the age is actually an integer? Like David has done in Node - https://github.com/getyoti/yoti-node-sdk/blob/development/src/dynamic_sharing_service/policy/dynamic.policy.builder.js#L68. 

Core will prevent this, but we might as well also add a check here, since I can't envisage this ever changing

Reply:
Age doesn't have to be an integer. "18" is a sensible input value too IMO.

Reply:
It is, I wouldn't have thought Core would handle that request though after they've implemented the changes. 

Additionally, as much as possible we want the SDKs to all behave in the same way. Since we only allow integers in the other SDKs, we should try to make this the same 👍